js/pages/tola_management_pages/country/models.js

```diff
@@ -200,16 +200,27 @@ export class CountryStore {
     }
 
     onSaveSuccessHandler({retroProgramCount}={}) {
-        let message = gettext("Successfully saved")
         if (retroProgramCount) {
-            message = interpolate(ngettext(
+            const message = interpolate(ngettext(
```

Comment:
maybe commenting in here as to why the save success happens two different ways (pnotify success [which is a top right growler, right?] vs our unified changeset) just to prevent a future dev from thinking that needs to be converted over